src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -563,311 +574,270 @@ const HomePage: React.FC = () => {
+                    )
+                  }
+                })}
 
-          {filteredDashboardList && filteredDashboardList.length > 0 && layoutView === LayoutViews.LIST && (
-            <Container className={css.masonry}>
-              <TableV2<DashboardInterface> className={css.table} columns={columns} data={filteredDashboardList || []} />
+              {tagsList?.resource?.tags?.length === 0 && <span>{getString('dashboards.homePage.noTags')}</span>}
             </Container>
-          )}
+          </Layout.Vertical>
+        </Container>
+      </Layout.Vertical>
 
-          {filteredDashboardList && filteredDashboardList.length === 0 && !loading && (
-            <Container style={{ height: 'calc(100vh - 226px)' }} flex={{ align: 'center-center' }}>
-              <Layout.Vertical spacing="medium" width={470} style={{ alignItems: 'center', marginTop: '-48px' }}>
-                <Icon name="dashboard" color={Color.GREY_300} size={35} />
-                <Heading level={2} font={{ align: 'center' }} color={Color.GREY_500}>
-                  {getString('dashboards.homePage.noDashboardsAvailable')}
-                </Heading>
-              </Layout.Vertical>
-            </Container>
+      <Layout.Vertical
+        padding="large"
+        style={{ width: 'calc(100% - 280px)', height: 'calc(100vh - 226px)', paddingTop: 0, overflow: 'scroll' }}
+      >
+        <Layout.Horizontal style={{ padding: 'var(--spacing-6) var(--spacing-9) ' }}>
+          <ExpandingSearchInput
+            placeholder={getString('dashboards.homePage.searchPlaceholder')}
+            onChange={(text: string) => {
+              setSearchTerm(text)
+            }}
+            className={css.search}
+          />
+        </Layout.Horizontal>
+        <Layout.Horizontal style={{ marginLeft: 'var(--spacing-xxxlarge)', alignItems: 'baseline' }}>
+          <section className={css.filteredTags}>
+            {filteredTags.map((tag: string, index: number) => {
+              return (
+                <section className={css.customTag} key={tag + index}>
+                  {tag}{' '}
+                  <Icon
+                    name="cross"
+                    style={{ cursor: 'pointer' }}
+                    onClick={() => {
+                      const filterTags = filteredTags.filter(v => v !== tag)
+                      setFilteredTags(filterTags)
+                    }}
+                  />
+                </section>
+              )
+            })}
+          </section>
+          {filteredTags?.length > 0 && (
+            <Text
+              color={Color.PRIMARY_7}
+              style={{ cursor: 'pointer' }}
+              font={{ weight: 'semi-bold' }}
+              onClick={() => setFilteredTags([])}
+            >
+              Clear All
+            </Text>
           )}
-        </Layout.Vertical>
+        </Layout.Horizontal>
+        {filteredDashboardList && filteredDashboardList.length > 0 && layoutView === LayoutViews.GRID && (
```

Comment:
You can simplify with optional chaining
```suggestion
        {!!filteredDashboardList?.length && layoutView === LayoutViews.GRID && (
```